### PR TITLE
Regenerate APIs and release three beta packages

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

**Comment by jskeet** [comment]
(Note that even after this is approved, I'm going to until I've got internal approval to merge/release the Language package.)

**Comment by jskeet** [comment]
Will look into the Travis failure. Looks like it's relying on a new common proto that we don't have yet.

File: apis/Google.Cloud.Redis.V1/Google.Cloud.Redis.V1/CloudRedisGrpc.cs [hunk 1]
```diff
@@ -536,142 +486,6 @@ protected CloudRedisClient(ClientBaseConfiguration configuration) : base(configu
...
-      ///
-      /// The returned operation is automatically deleted after a few hours, so
-      /// there is no need to call DeleteOperation.
-      /// </summary>
-      /// <param name="request">The request to send to the server.</param>
-      /// <param name="headers">The initial metadata to send with the call. This parameter is optional.</param>
-      /// <param name="deadline">An optional deadline for the call. The call will be cancelled if deadline is hit.</param>
-      /// <param name="cancellationToken">An optional token for canceling the call.</param>
-      /// <returns>The response received from the server.</returns>
-      public virtual global::Google.LongRunning.Operation ImportInstance(global::Google.Cloud.Redis.V1.ImportInstanceRequest request, grpc::Metadata headers = null, global::System.DateTime? deadline = null, global::System.Threading.CancellationToken cancellationToken = default(global::System.Threading.CancellationToken))
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Redis.V1/Google.Cloud.Redis.V1/CloudRedisGrpc.cs]: Why are all these changes in a GA version of the API? (I know we've only released a beta, but this looks like a GAed api version)

File: apis/Google.Cloud.SecurityCenter.V1/Google.Cloud.SecurityCenter.V1/SecuritycenterService.cs [hunk 2]
```diff
@@ -1067,22 +1065,6 @@ public sealed partial class GroupAssetsRequest : pb::IMessage<GroupAssetsRequest
       }
     }
 
-    /// <summary>Field number for the "having" field.</summary>
```
Comment (chrisdunelm) [inline apis/Google.Cloud.SecurityCenter.V1/Google.Cloud.SecurityCenter.V1/SecuritycenterService.cs]: Again, this looks like a breaking change to a GAed version?

**Comment by jskeet** [comment]
Fixed the Redis failure - this required a manual tweak to the proto due to an unintented import; that's being fixed elsewhere.

**Pull request merged**